utils/grep.xml

```diff
@@ -3,36 +3,52 @@
 <interface uri="http://repo.roscidus.com/utils/grep" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
   <name>Grep</name>
   <summary xml:lang="en">Grep: print lines matching a pattern</summary>
-  <description xml:lang="en">Grep searches one or more input files for lines containing a match to a specified pattern. By default, grep prints the matching lines. </description>
-  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.ico" type="image/vnd.microsoft.icon"/>
+  <description xml:lang="en">A Windows port of GNU Grep.
+
+  The latest port available from GnuWin32 is of version 2.5.4, which
+  was released in Feb 2009, quite some time ago.  Also, it always
+  annoyed me that color highlighting of Grep matches is not available
+  on MS-Windows.  This port of the latest release 2.10 of Grep fixes
+  that.
+
+  To use Perl-style regular expression, Grep needs to be built with
+  libpcre, so you have this as a bonus below.
+  </description>
```

Comment:
Probably better to keep the original description, since this feed does not only provide Windows builds of grep.

Reply:
done